packages/client/src/components/SafeSettings.js

```diff
@@ -192,7 +192,9 @@ const RemoveSafeOwner = ({ web3, safeOwner, onCancel, onSubmit }) => {
   const [name, setName] = useState(safeOwner.name);
   const [address, setAddress] = useState(safeOwner.address);
   const [addressValid, setAddressValid] = useState(true);
-  const isFormValid =  name.trim().length > 0 && addressValid;
+  //LL: will validate name once name is persisted on the chain
```

Comment:
so that means a contract change? what's the plan for that work?

Reply:
Yes, that means we need to change the data structure of signers from {Address: Bool} (representing if the signer is verified) to {Address: SignerStruct} (SignerStruct contains multiple fields like name, isVerified, etc). I believe this change is necessary for the work moving forward to handle users identities. Regardless if we want to use an external api for user identities or not, signer object needs to be more descriptive. 

The work mainly lies in tweaking the MyMultiSig.cdc, I don't see any script/transactions or front-end being impacted so far.

Reply:
I see. Noticed that in this [Vessel PRD page](https://www.notion.so/VESSEL-Treasury-Management-49d5dcb8b03d41898d273af0f79dee87) it mentioned ```All naming and imagery added to a safe is saved locally. Nothing is shared amoong wallet owners on safe besides address.```, wondering if it's per design to not have name on chain? 🤔  @jacksonConrad any idea?

Reply:
Just to close on this thread, there has been offline discussion on this and confirmed it's expected to not have `name` stored on chain. In future we might consider store it in a service but for now it will be on local storage only.